client/src/telethon/_impl/client/events/queries.py

```diff
@@ -43,6 +46,16 @@ def data(self) -> bytes:
         assert self._raw.data is not None
         return self._raw.data
 
+    @property
+    def chat(self) -> Optional[Chat]:
+        """
+        The :term:`chat` when the message was sent.
+        Only available if the event was triggered by a button under a usual message, not an inline one.
```

Comment:
```suggestion
        The :term:`chat` where the button was clicked.

        This will be :data:`None` if the message with the button was sent from a user's inline query.
```